.github/workflows/puppet-test.yml

```diff
@@ -95,3 +91,46 @@ jobs:
         with:
           name: puppet-${{ matrix.PUPPET_RELEASE }}-${{ matrix.PACKAGE_TYPE }}-${{ matrix.TEST_CASE }}-result
           path: ${{ env.RESULT_PATH }}
+
+  puppet-test-windows:
+    name: puppet-test-windows
+    runs-on: ${{ matrix.OS }}
+    needs:
+      - puppet-lint
+      - puppet-rake-spec
+    strategy:
+      matrix:
+        OS: [ "windows-2019", "windows-2022" ]
+        PUPPET_RELEASE: [ "6.0.2", "7.21.0" ]
+        TEST_CASE: [ "default", "custom_vars" ]
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v3
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          cache: pip
+          cache-dependency-path: ${{ env.REQUIREMENTS_PATH }}
+
+      - name: Install pytest
+        run: pip install -r "${{ env.REQUIREMENTS_PATH }}"
+
+      - name: Test puppet deployment
+        timeout-minutes: 45
+        run: |
+          $env:PUPPET_RELEASE = "${{ matrix.PUPPET_RELEASE }}"
+          mkdir ${{ env.RESULT_PATH }}
+          pytest --verbose -m windows `
```

Comment:
Do you know what this backtick line continuation is called or how it's provided? I'm not sure I've seen it before.

Reply:
I think it's just powershell's multiline command separator: https://shellgeek.com/powershell-multiline-command/#PowerShell_Multiline_Command